app/grandchallenge/algorithms/migrations/0046_alter_job_public.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.1.11 on 2023-10-05 08:56
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("algorithms", "0045_optionalhangingprotocolalgorithm_and_more"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="job",
+            name="public",
+            field=models.BooleanField(
+                default=False,
+                help_text="If True, allow anyone to download this result along with the input image. Otherwise, only the job creator will have permission to download and view this result.",
```

Comment:
For changes to help text you do not need to create a migration, find the last migration where this field was changed and change it there instead (Algorithms Migration 0001 Line 281)

Reply:
Done!